lib/routes/validators.js

```diff
@@ -77,8 +80,8 @@ module.exports.email = function() {
 module.exports.service = isA.string().max(16).regex(/^[a-zA-Z0-9\-]*$/)
 module.exports.hexString = isA.string().regex(HEX_STRING)
 module.exports.clientId = module.exports.hexString.length(16)
-module.exports.accessToken = module.exports.hexString.length(32)
-module.exports.refreshToken = module.exports.hexString.length(32)
+module.exports.accessToken = module.exports.hexString.length(64)
```

Comment:
Does this mean a max length of 64 or that it always must be 64? If it must be 64, does that mean existing access/refresh tokens cause a blowup?

Reply:
> If it must be 64, does that mean existing access/refresh tokens cause a blowup?

Nah, I think this was just a typo, and this was never used